### PR TITLE
motor monitoring and vibration Notecard only app

### PR DESCRIPTION
**Pull request by m-mcgowan**
# Problem Context

## Testing

Unit / integration / e2e tests?
Steps to test manually?
Any other sorts of testing notes to include?

### CODE REVIEW

File: motor_monitor_and_vibration/README.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+In these steps, you will use the in-browser terminal to configure your Notecard to report the motor state and motion events to your Notehub project. You may also skip these individual steps and use the [#Notecard-CLI] to configure the Notecard using a single command.
+
+### Firmware
+
+The Notecard should use [firmware version 3.5.1](https://dev.blues.io/notecard/notecard-firmware-updates/#v3-5-1-october-7th-2022) or higher version. The simplest way to update firmware is to do an [over-the-air update](https://dev.blues.io/notecard/notecard-firmware-updates/#ota-dfu-with-notehub).
+
+### Configure Notehub Project and Connection Mode
+
+Set the ProductUID for the Notecard by pasting the command below into the in-browser terminal. Make sure to replace `com.your-company:your-product-name` with the ProductUID from your Notehub project, which can be found below your project's name in the dashboard at https://notehub.io. Also, replace the placeholder serial number (`"sn:" "motor-location"`) with the location of your fridge (e.g. "kitchen").
```
Comment (haydenroche5) [inline motor_monitor_and_vibration/README.md:67]: Serial number part needs updating. Looking like it's the snippet from the fridge monitor's README.md.

File: motor_monitor_and_vibration/README.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+This app uses `periodic` mode to minimize power use so that the solution can remain online for longer when mains power fails. `"outbound": 5` is the max wait time, in minutes, to sync outbound data from the Notecard. For more details, see [our Essential Requests documentation for `hub.set`](https://dev.blues.io/notecard/notecard-walkthrough/essential-requests/#notehub-configuration).
+
+### Configure Vibration Sensing
+
+Vibration sensing is configured using a number of distinct requests. Each request given below should be pasted into the in-browser terminal separately.
+
+#### `card.motion.mode`
+
+The [`card.motion.mode` requst](https://dev.blues.io/reference/notecard-api/card-requests/#card-motion-mode) enables motion sensing using the Notecard's built-in accelerometer. Motion is used to deterrmine how much the motor is vibration. The configuration below uses maximum sensitivity (`5`) and a 5 second period for each bucket of movement count.
```
Comment (haydenroche5) [inline motor_monitor_and_vibration/README.md:81]: requst -> request and maybe move the "request" part out of the hyperlink. It's not included on the others below.

deterrmine -> determine

"Motion is used to deterrmine how much the motor is vibration" think this sentence needs reworking.

File: motor_monitor_and_vibration/README.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+  "req":       "card.motion.track",
+  "minutes":   5,
+  "count":     60,
+  "start":     true
+}
+```
+
+### `card.motion.sync`
+
+THe [`card.motion.sync`](https://dev.blues.io/reference/notecard-api/card-requests/#card-motion-sync
```
Comment (haydenroche5) [inline motor_monitor_and_vibration/README.md:108]: THe -> The

File: motor_monitor_and_vibration/README.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+  "start":    true,
+  "minutes":  5
+}
+```
+
+### `card.aux - gpio mode`
+
+The [`card.aux`](https://dev.blues.io/notecard/notecard-walkthrough/advanced-notecard-configuration/#monitoring-aux-gpio-state-with-environment-variables) request configures how the Notecard uses the auxilliary GPIO pins.  Here we wish to either monitor or control the motor - which you use depends upon whether the motor provides the activity singal or whether the Notecard controls the motor's activity.
+
+When the motor privdes an activity signal, configure `AUX1` as a input.
```
Comment (haydenroche5) [inline motor_monitor_and_vibration/README.md:124]: privdes -> provides

File: motor_monitor_and_vibration/README.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+
+When the motor should be controlled from the Notecard's `AUX1` pin, set the pin as an output, initially low.
+
+```json
+{"req":"card.aux", "mode":"gpio", "usage": [
+  "low"
+]}
+```
+
+Next we enable GPIO monitirong or control via environment variables, and sync changes to notehub.
```
Comment (haydenroche5) [inline motor_monitor_and_vibration/README.md:140]: monitirong -> monitoring

File: motor_monitor_and_vibration/README.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+
+```sh
+notecard -setup config.json
+```
+
+You may also make other edit to the configuration file to change
+* vibration sensitivity (`sensitivity` property in the `card.motion.mode` command)
+* the number of buckets of vibration data captured
+* how often motion events are captured on the Notecard
+* how often events are send to Notehub
```
Comment (haydenroche5) [inline motor_monitor_and_vibration/README.md:160]: send -> sent

File: motor_monitor_and_vibration/README.md [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+* change how often motion events are sent to the cloud
```
Comment (haydenroche5) [inline motor_monitor_and_vibration/README.md:161]: Is this different from the previous bullet point?

File: motor_monitor_and_vibration/README.md [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+
+## Create a Route with JSONata
+
+The app uses a JSONata in a Notehub route to determine the amount of vibration and to signal an alert when vibration is out of range for the current state of the motor.
+
+1. Create a new route with notefile `_notion.qo`. Copy the contents of the `route.jsonata` from this project into the JSONata transformation text input in the route setup page.
```
Comment (haydenroche5) [inline motor_monitor_and_vibration/README.md:167]: notion -> motion

**Review by haydenroche5 — CHANGES_REQUESTED** [review]

File: motor_monitor_and_vibration/README.md (hunk 2 above)
Comment (m-mcgowan) [reply motor_monitor_and_vibration/README.md:81]: I need a spell checker in VSCode :-)

File: motor_monitor_and_vibration/README.md [hunk 9]
```diff
@@ -0,0 +1,186 @@
...
+
+Sign up for a free account on [Notehub.io](https://notehub.io) and create a new project.
+
+## Notecard Setup
+
+In these steps, you will use the in-browser terminal to configure your Notecard to report the motor state and motion events to your Notehub project. You may also skip these individual steps and use the [#Notecard-CLI] to configure the Notecard using a single command.
+
+### Firmware
+
+The Notecard should use [firmware version 3.5.1](https://dev.blues.io/notecard/notecard-firmware-updates/#v3-5-1-october-7th-2022) or higher version. The simplest way to update firmware is to do an [over-the-air update](https://dev.blues.io/notecard/notecard-firmware-updates/#ota-dfu-with-notehub).
```
[the remark this answers is not in the document]
Comment (m-mcgowan) [reply motor_monitor_and_vibration/README.md:63]: lol we had this one before, and I thought it went this way.  removed extraneous "version".

File: motor_monitor_and_vibration/README.md (hunk 2 above)
Comment (m-mcgowan) [reply motor_monitor_and_vibration/README.md:81]: vibration -> vibrating. 

This plugin works well. https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

File: motor_monitor_and_vibration/README.md [hunk 10]
```diff
@@ -0,0 +1,184 @@
...
+{
+  "req":      "card.motion.sync",
+  "start":    true,
+  "minutes":  5
+}
+```
+
+### `card.aux - gpio mode`
+
+The [`card.aux`](https://dev.blues.io/notecard/notecard-walkthrough/advanced-notecard-configuration/#monitoring-aux-gpio-state-with-environment-variables) request configures how the Notecard uses the auxillary GPIO pins.  Here we wish to either monitor or control the motor - which you use depends upon whether the motor provides the activity signal or whether the Notecard controls the motor's activity.
```
Comment (haydenroche5) [inline motor_monitor_and_vibration/README.md:121]: Looks like auxillary is still misspelled.

**Review by haydenroche5 — CHANGES_REQUESTED** [review]
I'm no JSONata expert, but everything else is looking good. :)

File: motor_monitor_and_vibration/README.md (hunk 10 above)
Comment (m-mcgowan) [reply motor_monitor_and_vibration/README.md]: l -> i done :-)

**Review by haydenroche5 — APPROVED** [review]

**Comment by m-mcgowan** [comment]
[NF20 implementation](https://trello.com/c/RDODaWnk/156-nf20-implementation)

**Pull request merged**